src/backend/MoveList.js

```diff
@@ -5,64 +5,56 @@ import {firestore, root} from "../config/firebase";
 import firebase from 'firebase';
 
 
-export async function MoveList(props){
+export function MoveList(props){
 	const [assassinEnabled, setAssassin] = useState(true);
 	const [coupEnabled, setCoup] = useState(true);
 	const [theRest, setTheRest] = useState(false);
+	const [coins, setCoins] = useState(0);
 
-	await firestore.collection(root).doc(props.roomName).collection("players").doc(props.activePlayerID).get().then((player)=>{
-		if (player.data().coins >= 3){
-			setAssassin(false)
-		};
-		if (player.data().coins >= 7){
-			setCoup(false)
-		};
-		if (player.data().coins >= 10){
-			setTheRest(true)
-		};
+	useEffect( () => {
+		firestore.collection(root).doc(props.roomName).collection("players").doc(props.activePlayerID).get().then((player)=>{
+			console.log("HEEEEEEEEEEEEEEEEE");
```

Comment:
lmoa my bad i forgot to take out my debug logs, can you remove them from your brach after the merge

Reply:
I am definitely NOT trying to improve my LoC

Reply:
Sure ... 🙄

Reply:
LOL